app/client/components/billing/billing.tsx

```diff
@@ -100,6 +114,24 @@ const BillingRenderer = (apiResponse: MembersDataApiItem[]) => {
                     null,
                     !!productDetail.alertText
                   );
+                  const paidPlan = getMainPlan(
+                    productDetail.subscription
+                  ) as PaidSubscriptionPlan;
+                  const productInvoiceData = invoiceData
+                    .filter(
+                      invoice =>
+                        invoice.subscriptionName ===
```

Comment:
It's a shame the two APIs can't agree on a name for this field. In the Zuora UI it's Subscription Number, in their API it's subscriptionName, in our models and UX it's subscriptionId and Subscription ID respectively 🤦

Reply:
Hey Paul, yep you are right. It's been discussed and there is a plan to take a look at members data and MMA with the goal of renaming `subscriptionId` to `subscriptionName` it a separate ticket.